variant.go

```diff
@@ -50,15 +50,15 @@ type Variant struct {
 	Option3              string           `json:"option3,omitempty"`
 	CreatedAt            *time.Time       `json:"created_at,omitempty"`
 	UpdatedAt            *time.Time       `json:"updated_at,omitempty"`
-	Taxable              bool             `json:"taxable,omitempty"`
+	Taxable              *bool            `json:"taxable,omitempty"`
```

Comment:
Please could you add a unit test for this change?

Reply:
Done - let me know what you think.